execute/table.go

```diff
@@ -754,6 +757,7 @@ type ColListTableBuilder struct {
 	key     flux.GroupKey
 	colMeta []flux.ColMeta
 	cols    []columnBuilder
+	nils    []map[int]bool
```

Comment:
This would be much more efficient and easier to follow if it was a `map[struct{i ,j int}]bool` I know I said that this is basically throw away code but managing a list of maps is cumbersome vs just a single map.

Reply:
I personally find the list easier because the thing that builds the array goes column by column.